### PR TITLE
Feat account

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Added dropdown menu for account options
- Can Log In and Sign Up
- Can log out from the nav when in wish-list and cart page

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -51,6 +51,41 @@ header {
...
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    top: 3rem;
+    background-color: var(--light-grey);
+    min-width: 160px;
```
Comment (Shivanshsharma01) [inline css/navbar.css:71]: can use rem here

**Review by Shivanshsharma01 — APPROVED** [review]
Clean and structured code, Good use of variables throughout. Can make the suggested change

File: css/navbar.css (hunk 1 above)
Comment (hinanshisuthar) [reply css/navbar.css:71]: Yes, true. But here I didn't wanted it to be responsive hence used px instead of rem.

**Review by ZuberDunge — APPROVED** [review]
Well structured and readable code. Everything looks fine.  Good to go from my side.

**Review by stutisk — APPROVED** [review]
Looks good to me ! Can go forward with merging it

**Pull request merged**